assets/src/edit-story/masks/.eslintrc

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
Is this obsolete now or just deleted by accident?

Reply:
obsolete, moved the rule inside /output.js